src/test/groovy/org/prebid/server/functional/tests/privacy/GppTransmitUfpdActivitiesSpec.groovy

```diff
@@ -0,0 +1,810 @@
+package org.prebid.server.functional.tests.privacy
+
+import org.prebid.server.functional.model.bidder.BidderName
+import org.prebid.server.functional.model.bidderspecific.BidderRequest
+import org.prebid.server.functional.model.db.StoredRequest
+import org.prebid.server.functional.model.request.auction.Activity
+import org.prebid.server.functional.model.request.auction.ActivityRule
+import org.prebid.server.functional.model.request.auction.AllowActivities
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.Condition
+import org.prebid.server.functional.model.request.auction.Data
+import org.prebid.server.functional.model.request.auction.Device
+import org.prebid.server.functional.model.request.auction.Eid
+import org.prebid.server.functional.model.request.auction.User
+import org.prebid.server.functional.model.request.auction.UserExt
+import org.prebid.server.functional.model.request.auction.UserExtData
+import org.prebid.server.functional.model.request.amp.AmpRequest
+import org.prebid.server.functional.util.PBSUtils
+
+import static org.prebid.server.functional.model.bidder.BidderName.GENERIC
+import static org.prebid.server.functional.model.bidder.BidderName.OPENX
+import static org.prebid.server.functional.model.request.auction.ActivityRule.Priority.DEFAULT
+import static org.prebid.server.functional.model.request.auction.ActivityRule.Priority.HIGHEST
+import static org.prebid.server.functional.model.request.auction.ActivityRule.Priority.INVALID
+import static org.prebid.server.functional.model.request.auction.ActivityType.TRANSMIT_UFPD
+import static org.prebid.server.functional.model.request.auction.Condition.ConditionType.BIDDER
+import static org.prebid.server.functional.model.request.auction.Condition.ConditionType.GENERAL_MODULE
+import static org.prebid.server.functional.model.request.auction.Condition.ConditionType.RTD_MODULE
+import static org.prebid.server.functional.model.request.auction.DistributionChannel.SITE
+
+class GppTransmitUfpdActivitiesSpec extends PrivacyBaseSpec {
+
+    private String activityProcessedRulesForAccount
+    private String disallowedCountForAccount
```

Comment:
1.Please use this variables inside tests or make template `"accounts.%s.activity.processedrules.count"` and then replace value.